### DISCUSSION TITLE
Restyle vs  tailwindcss

### DISCUSSION THREAD

**Discussion by maniish** [obytes/react-native-template-obytes#214] [Q&A] [answered]
I can see you have used restyle but then switched to tailwinds. Is there any particular reason for doing it

**Comment by yjose** [comment, marked as answer]
@maniish Yeah, that's correct. We used `restyle` with older versions of the template, and we switched to Tailwind after trying NativeWind for a couple of projects. Here are some of the reasons why we switched to Tailwind:

* Most of our team members are familiar with Tailwind CSS, and we have noticed that our team is more productive with it, especially since most of them come from a web background and are used to using it.
* Sometimes, with`restyle`, we end up with elements that have 10 properties just for styling, which is not good, to be honest, compared to using a single `className` prop. 
* More flexibility with tailwindCSS in case we want to support more complex variant

But overall, "restyle" is an excellent solution too, and we still use it for some old projects. However, for new projects, we are using Tailwind CSS.

**Comment by spumony** [comment]
Hi there, 
Nice template and perfect documentation. Thank you
Can I find the project without any UI library? 😄
